### ISSUE TITLE
Help with installation 

### ISSUE THREAD

**Issue by XDe0** [dxxxxy/EssentialCosmeticsUnlocker#5]
I'm trying to use this mod, but it's not working. I saw #4 and did same, I put `ecu-1.2.1.jar` and `Essential-fabric_1-20-1.jar` in `.minecraft/mods` also I have `.minecraft/essentials` folder, but in Essential wardrobe I still see a price and don't have these items. If needed, I have **1.20.1 Minecraft version** and play with **MultiMC client**
Do you know what to do?
[mods.zip](https://github.com/dxxxxy/EssentialCosmeticsUnlocker/files/12293657/mods.zip)

**Comment by dxxxxy** [comment]
https://github.com/dxxxxy/EssentialCosmeticsUnlocker#notes

You're not placing the right fabric mod.

**Comment by XDe0** [comment]
Ummmm, sorry, but what now I need to do, cuz I'm blunt in this topic. I need to install legacy or older version?

**Comment by dxxxxy** [comment]
You need to read what I've linked.

**Comment by XDe0** [comment]
Um, I read Notes from A to Z but still any works. I used `-legacy.jar` tried to move files in different folders. But I still don't understand

**Comment by dxxxxy** [comment]
You're using Fabric 1.20. Legacy has nothing to do here. Check the table and read it again.

**Comment by XDe0** [comment]
Oh sorry, so I need to install Fabric on 1.20.0
But when will u update your **ecu**?

**Comment by dxxxxy** [comment]
`Essential-fabric_1-20-1.jar` is not the right mod. Reread the note with a * again.

**Comment by XDe0** [comment]
Sorry, but I still don't understand what should I do

**Comment by dxxxxy** [comment]
Ummm, I don't really know what to say since its said word by word but you are not placing the right Essential mod (I can tell by the name). It's written in Notes and I cant really explain in simpler terms.

**Comment by XDe0** [comment]
OMG. I'm apologizing for whole issue, i tried to install Essential with built-in installer, created new profile and added `Essential (fabric_1.20.1).jar` and `ecu-1.2.1.jar` in `.minecraft/mods` and now I have all Essential items. Thank you very much for help!
